### ISSUE TITLE
Can you add a function to scroll to a certain index?

### ISSUE THREAD

**Issue by CoderWeiLee** [GONZALEZD/cupertino_listview#10]
### This is very useful, I now try to use `GlobalKey `to achieve this function,

`Scrollable.ensureVisible(key2.currentContext!,duration: const Duration(seconds: 1), curve: Curves.easeInOut);` but  There seems to be a problem with this writing, the key3 bound to the off-screen Widget will report `context == null`

**Comment by GONZALEZD** [comment]
Yes this case is very complex to solve in Flutter, as widget may not have context.
If you find a solution, please make a PR about it.
